functional/FunctionalTest.java

```diff
@@ -1125,8 +1127,14 @@ public static void getObject_test6() throws Exception {
         client.putObject(bucketName, objectName, is, 0, nullContentType);
       }
 
-      client.getObject(bucketName, objectName)
-          .close();
+      try {
+        client.getObject(bucketName, objectName).close();
+      } catch (final ErrorResponseException e) {
+        // current implementation results in request headers="Range: bytes=0-"; response is a 416 with the given message
```

Comment:
We can remove this comment, this is an expected response.

Reply:
removed